tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "CommonJS", //fireship recomande NodeNext
+        // "moduleResolution": "NodeNext"
+        "esModuleInterop": true,
+        "allowSyntheticDefaultImports": true,
+        "target": "ES6",
+        "noImplicitAny": true,
+        "moduleResolution": "node",
+        "sourceMap": true,
+        "outDir": "dist",
+        "baseUrl": ".",
+        "paths": {
+            "*": [
+                "node_modules/*",
+                "src/types/*"
+            ]
```

Comment:
En nice to have, tu pourras faire des paths alias
https://dev.to/larswaechter/path-aliases-with-typescript-in-nodejs-4353